packages/databricks-vscode/src/configuration/ConfigureWorkspaceWizard.ts

```diff
@@ -194,10 +194,7 @@ export class ConfigureWorkspaceWizard {
         if (pick.profile !== undefined) {
             // TODO: If the profile has an auth type with deeper support in the Extension, (azure-cli, databricks-cli),
```

Comment:
We should track this TODO and at least cover Azure CLI and Databricks CLI auth

Reply:
Might as well fix it :). 
https://github.com/databricks/databricks-vscode/pull/1061